src/pip/_internal/wheel.py

```diff
@@ -163,13 +163,15 @@ def message_about_scripts_not_on_PATH(scripts):
         grouped_by_dir[parent_dir].add(script_name)
 
     # We don't want to warn for directories that are on PATH.
-    not_warn_dirs = os.environ["PATH"].split(os.pathsep)
+    not_warn_dirs = list(
+        os.path.normcase(i) for i in os.environ["PATH"].split(os.pathsep)
```

Comment:
`normcase` also converts slashes to backslashes on Windows, so this should also catch `/` vs `\` discrepancies (which I included in my test code).

Reply:
Awesome. I'll add the test. :)